packages/fuels/tests/harness.rs

```diff
@@ -1723,35 +1724,33 @@ async fn test_arrays_with_custom_types() -> Result<(), Error> {
     assert_eq!(states[1], response.value[1]);
     Ok(())
 }
+// TODO: Enable test
```

Comment:
Is this TODO as in I'll do it later in my PR or is there something stopping you from doing it in this PR?

IIRC these are the ones that required the new sway stdlib to work? Isn't that out by now? Seem to remember @digorithm enabling some tests.